client/network-gossip/src/state_machine.rs

```diff
@@ -544,22 +526,6 @@ mod tests {
 		assert_eq!(consensus.messages.len(), 2);
 	}
 
-	#[test]
-	fn can_keep_multiple_subscribers_per_topic() {
```

Comment:
This has been moved to `bridge.rs` and renamed to `keeps_multiple_subscribers_per_topic_updated_with_both_old_and_new_messages`.